src/OpenTelemetry.Extensions.Enrichment/OpenTelemetry.Extensions.Enrichment.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimumSupportedVersion)</TargetFrameworks>
+    <Description>OpenTelemetry .NET SDK telemetry enrichment.</Description>
+    <NoWarn>$(NoWarn),CS1591</NoWarn>
+    <MinVerTagPrefix>Extensions.Enrichment-</MinVerTagPrefix>
+    <Nullable>enable</Nullable>
+    <EnableAnalysis>true</EnableAnalysis>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="7.0.2" />
```

Comment:
If you guys want to use this, I'm fine with it. But personally, I stay away from this package. It was not taken into dotnet\runtime so it doesn't carry the strong guarantees we rely on. [I was also told it is infrastructure only for AspNetCore](https://github.com/dotnet/aspnetcore/issues/36586#issuecomment-922063322).

Buyer beware 😄

Reply:
As I understand from the thread this is available everywhere so we can use it new features won't be added to make it more convenient for non AspNetCore. We use the pools everywhere in R9 libraries (not just telemetry) today and there hasn't been any issues. 

If something comes up in future and we don't get the support from .NET we can always replace it with our custom pooling implementation, but I think at this point I don't see a reason not to leverage it.

Reply:
Agree with Deepaks. Lets tart with MS.Ext.ObjectPool here.